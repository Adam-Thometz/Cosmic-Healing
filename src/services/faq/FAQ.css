.FAQ {
  margin: var(--margin) 0;
  animation: slide-up var(--duration) calc(var(--delay) * var(--i)) both;
}