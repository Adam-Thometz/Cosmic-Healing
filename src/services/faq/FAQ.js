import React from 'react';

import SectionHeader from '../../_components/section-header/SectionHeader';
import SmallCaption from '../../_components/small-caption/SmallCaption';
import Question from './question/Question';

import questionsAnswers from './questionsAnswers';

const FAQ = () => {
  const faq = questionsAnswers.map(({ question, answer }) => (
    <Question question={question} answer={answer} />
  ));
  
  return (
    <section className='FAQ'>
      <SmallCaption text="FAQ" />
      <SectionHeader text="Frequently Asked Questions" />
      {faq}
    </section>
  );
};

export default FAQ;