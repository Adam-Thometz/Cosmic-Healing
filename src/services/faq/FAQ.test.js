import React from "react";
import { render } from "@testing-library/react";
import FAQ from './FAQ';

describe('FAQ component', () => {
  it('renders without crashing', () => {
    render(<FAQ />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<FAQ />);
    expect(asFragment()).toMatchSnapshot();
  });
});