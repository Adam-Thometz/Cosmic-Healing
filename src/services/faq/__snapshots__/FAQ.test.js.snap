// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQ component matches the snapshot 1`] = `
<DocumentFragment>
  <section
    class="FAQ"
  >
    <small
      class="SmallCaption"
      style="color: black;"
    >
      FAQ
    </small>
    <h2
      class="SectionHeader"
    >
      Frequently Asked Questions
    </h2>
    <div
      class="Question"
    >
      <div
        class="Question-question-wrapper"
      >
        <h3
          class="SectionHeader"
        >
          How much does a session cost?
        </h3>
        <img
          alt="Pick to expand question"
          src="expanded.png"
        />
      </div>
    </div>
    <div
      class="Question"
    >
      <div
        class="Question-question-wrapper"
      >
        <h3
          class="SectionHeader"
        >
          Where are sessions held?
        </h3>
        <img
          alt="Pick to expand question"
          src="expanded.png"
        />
      </div>
    </div>
    <div
      class="Question"
    >
      <div
        class="Question-question-wrapper"
      >
        <h3
          class="SectionHeader"
        >
          What is your deposit policy?
        </h3>
        <img
          alt="Pick to expand question"
          src="expanded.png"
        />
      </div>
    </div>
    <div
      class="Question"
    >
      <div
        class="Question-question-wrapper"
      >
        <h3
          class="SectionHeader"
        >
          What insurance do you accept?
        </h3>
        <img
          alt="Pick to expand question"
          src="expanded.png"
        />
      </div>
    </div>
  </section>
</DocumentFragment>
`;
